include/regs.h

```diff
@@ -64,7 +64,7 @@
 #define R_AT_LERP_MIN               OREG(41)
 #define R_AT_LERP_SCALE             OREG(42)
 #define R_CAM_YOFFSET_NORM          OREG(46)
-#define R_CAM_FIXED3_FOV            PREG(7)
+#define R_CAM_DATA(data)            PREG(data)
```

Comment:
Indexing data by an argument called `data` feels weird, maybe it should just be `type`?
```suggestion
#define R_CAM_DATA(type)            PREG(type)
```